script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
```

Comment:
Consider using `yaml.safe_load()` directly on the file object for better efficiency and readability.
```suggestion
        print(yaml.dump(yaml.safe_load(yaml_file), indent=2))
```